windtunnel/pre_processing.py

```diff
@@ -110,6 +110,14 @@ def compute_object_length(mesh: pv.PolyData):
 
 
 def save_mesh_obj(mesh, dest_object_path):
+    faces = mesh.faces
```

Comment:
if the number of faces is divisible by 12 (3*4) this logic might fail to do what we expect...